data/starts.txt

```diff
@@ -0,0 +1,186 @@
+	choice
+		`	(Go to the shipyard.)`
+			goto shipyard
+		`	(Go see what the captain has to offer first.)`
+	`	You head to the spaceport and easily find the captain, a Hai, selling one of his Centipedes to another Hai. Thankfully it looks like he has not sold the Grasshopper yet. They happen to be conversing in your language, and you can overhear part of the conversation as the buyer is speaking. "... great that some people still care about helping others make their way to the stars faster, instead of only looking at profit."`
+	`	He responds with a smile, "Buying back a ship for a fair price is how I got to the stars myself! It really is a great change of scenery, but I think I got to the point where I want to leave space for others in space, and help others on the ground instead. I am planning on teaching and learning now."`
+	`	"What a noble goal indeed," she says, before noticing you, "But it seems you have a new customer here. I will leave you to it." The captain turns to you, "Hello, good fellow! I assume you come for the Grasshopper."`
+	choice
+		`	"Indeed."`
+		`	"How did you know?"`
+		`	"Sorry, not interested."`
+			decline
+	`	"I can see the will to explore in your eyes, and with your age it just seems like the perfect fit. I would be delighted to have you flying the 'Pristine Jewel'!" he says, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
+	scene "thumbnail/hai grasshopper"
+	`	"Because I like you I'll give you a friendly price. What do you say about... a bit over a million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
+	choice
+		`	"Sold!"`
+		`	"Sorry, not interested. I'll go to the shipyard instead."`
+			goto shipyard
+	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. In your excitement to buy the ship all you can hear from the Hai captain's speech is the repeated reference to the Grasshopper's 'exceptional' reinforced armor.`
+	action
+		set "hai space start: Grasshopper"
+		give ship "Grasshopper (Expensive)" "Pristine Jewel"
+		payment -1132000
+	`	Before you leave, he gives you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`
+		decline
+	label shipyard
+	`	The pictures and videos you've seen of Hai ships before simply don't do them justice; seeing them in all their splendor here in the shipyard is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look.`
```

Comment:
The player’s never seen a Hai ship near the equator before?

Reply:
Yeah they've never been to the shipyards

Reply:
But Hai ships would have landed at the town they live at, right?

Reply:
I don't think so, you'd need a spaceport and it isnt that big
this planet has humans living near the equator and hai at the poles

Reply:
OK. This is resolved than.